Update bitbucket-server.mdx (#62)
hashicorp/terraform-docs-common: website/docs/cloud-docs/vcs/bitbucket-server.mdx

   ![Bitbucket Server screenshot: The application links page](/img/docs/bitbucket-server-application-links.png)

1. In application links, select Create link.

> 2. Select Atlassian product as the link type. This option also works for external applications. This is the original application links mechanism – Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.

Reviewer: I suggest rewording to "This method is the original application links mechanism. Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.

> 1. In application links, select Create link.
> 2. Select Atlassian product as the link type. This option also works for external applications. This is the original application links mechanism – Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.

Reviewer 2: I'm wondering if we can shorten that even more. I want to phrase this in terms of what this lets users do and address the user as "you" rather than talking about "users" more broadly. Thoughts? 

@mgarrell777 - Feel free to weigh in here too! I took you original edit above and put it here. :) 
```suggestion
1. Select **Create link** in application links.
2. Select **Atlassian product** as the link type. This option also works for external applications lets you continue to use OAuth 1.0 integrations.
```

> 3. Enter Terraform Cloud's URL in the text field (https://app.terraform.io, or the hostname of your Terraform Enterprise instance).

Reviewer: ```suggestion
3. Enter Terraform Cloud's URL in the text field (https://app.terraform.io or the hostname of your Terraform Enterprise instance).
```

Removal of an unneeded comma

>    ~> **Note:** If you're connecting multiple Terraform Cloud organizations to the same Bitbucket Server instance, you can only use Terraform Cloud's main URL once. For subsequent organizations, you can enter the organization URL instead. Organization URLs look like `https://app.terraform.io/app/<ORG NAME>` or `https://<TFE HOSTNAME>/app/<ORG NAME>` — it's the page Terraform Cloud's "Workspaces" button takes you to.

Reviewer: ```suggestion
   ~> **Note:** If you connect multiple Terraform Cloud organizations to the same Bitbucket Server instance, you can only use Terraform Cloud's main URL once. For subsequent organizations, you can enter the organization URL instead. Organization URLs are the pages you go to using Terraform Cloud's **Workspaces** button and look like `https://app.terraform.io/app/<ORG NAME>` or `https://<TFE HOSTNAME>/app/<ORG NAME>`.
```

Suggest rewording these sentences for clarity and to avoid contractions.
